apps/zbugs/schema.ts

```diff
@@ -189,26 +189,16 @@ type AuthData = {
   role: 'crew' | 'user';
 };
 
-export const schema = createSchema(
-  5,
-  {
-    user,
-    issue,
-    comment,
-    label,
-    issueLabel,
-    viewState,
-    emoji,
-    userPref,
-  },
-  {
+export const schema = createSchema(5, {
+  tables: [user, issue, comment, label, issueLabel, viewState, emoji, userPref],
+  relationships: [
```

Comment:
Same question here. what happens if two of these relationships objects have the same source? What if the same relationship name is declared twice?

Reply:
There's validation but it got added to the second commit rather than the first.  I miss mercurial's absorb command that automatically amends working changes to the correct commits :(